betydb2geojson.py

```diff
@@ -11,20 +11,29 @@
 ENV_BETYDB_URL_NAME = 'BETYDB_URL'
 
 
-def add_arguments(parser: argparse.ArgumentParser) -> None:
+def add_arguments():
     """Adds arguments to the command line parser
-    Arguments:
-        parser: instance of argparse.ArgumentParser to add to
     Return:
         No return is defined
     """
-    parser.add_argument('--betydb_url', help='the URL of BETYdb instance to query (defaults to ' + ENV_BETYDB_URL_NAME +\
-                                             ' environment variable)')
-    parser.add_argument('--filter', help='partial or full string filter for sitename values returned')
-    parser.add_argument("output_file", help='the output file to write GeoJSON to')
+    parser = argparse.ArgumentParser(description="BETYdb plots to GeoJSON",
+                                     formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('-u', '--betydb_url',
+                        help='the URL of BETYdb instance to query (defaults to ' + ENV_BETYDB_URL_NAME + \
+                             ' environment variable)', metavar='str', type=str, default=os.getenv('BETYDB_URL'))
+    parser.add_argument('-f', '--filter', help='partial or full string filter for sitename values returned',
+                        metavar='str', type=str, default='')
+    parser.add_argument('-o', '--outfile', help='the output file to write GeoJSON to', metavar='FILE', type=argparse.FileType('wt'),
+                        default='out.txt')
+
+    args = parser.parse_args()
+    if not args.betydb_url:
+        parser.error('--betydb_url is required')
 
+    return args
 
-def query_betydb_experiments(betydb_url: str = None) -> dict:
+
+def query_betydb_experiments(betydb_url: str) -> dict:
```

Comment:
```suggestion
def query_betydb_experiments(betydb_url: str = None) -> dict:
```
The original is correct, the parameter is not a requirement.